third-party/snyk/snapshot.json

```diff
@@ -112,14 +113,6 @@
       {"id":"npm:mustache:20151207","severity":"medium","semver":{"vulnerable":["<2.2.1"]}},
       {"id":"npm:mustache:20110814","severity":"medium","semver":{"vulnerable":["< 0.3.1"]}}
     ],
-    "next":[
```

Comment:
Any ideas why this is disappearing @lirantal? :)

AFAIK it should still be detectable (https://github.com/johnmichel/Library-Detector-for-Chrome/blob/b4f9a007addc19608a3864943aa735e8be6a6444/library/libraries.js#L449)

and the vuln still seems to exist on the [snyk site](https://snyk.io/vuln/SNYK-JS-NEXT-561584)

Reply:
Taking a look 🏎

Reply:
@patrickhulce this was an error on our side, probably due to usage of version 5.2.0 of js-library-detector. We now upgraded to 5.9.0 and these vulns are now back. We opened https://github.com/GoogleChrome/lighthouse/pull/10840 to verify the fix is working.

Reply:
great thanks for looking into it! 🙏